project/CodeCoverageSettings.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import sbt.Setting
+import scoverage.ScoverageKeys
+
+object CodeCoverageSettings {
+
+  private val excludedPackages: Seq[String] = Seq(
+    "<empty>",
+    "Reverse.*",
+    "uk.gov.hmrc.BuildInfo",
+    "app.*",
+    "prod.*",
+    ".*Routes.*",
+    "config.*",
+    "testOnly.*",
+    "testOnlyDoNotUseInAppConf.*"
+  )
+
+  val settings: Seq[Setting[_]] = Seq(
+    ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
+    ScoverageKeys.coverageMinimum := 95,
```

Comment:
`sbt clean coverage test it:test coverageReport` only gets 92.20%, so the build might fail here. We need to either test more stuff or lower this number... or set `coverageFailOnMinimum` to false (probably a bad idea)

Reply:
Will take a look at this.